common/src/main/java/com/wynntils/models/worlds/WorldStateModel.java

```diff
@@ -31,7 +31,7 @@
 public final class WorldStateModel extends Model {
     private static final UUID WORLD_NAME_UUID = UUID.fromString("16ff7452-714f-2752-b3cd-c3cb2068f6af");
     private static final Pattern WORLD_NAME = Pattern.compile("^§f {2}§lGlobal \\[(.*)\\]$");
-    private static final Pattern HOUSING_NAME = Pattern.compile("^§f  §l([^§\"\\\\]{1,18})$");
+    private static final Pattern HOUSING_NAME = Pattern.compile("^§f  §l([^§\"\\\\]{1,35})$");
```

Comment:
18 is the max limit that Wynn allows, but by default a guild housing name is "<guildname>'s HQ" and guilds have a name limit of 30 characters according to [the wiki](https://wynncraft.wiki.gg/wiki/Guilds#Creating)